### ISSUE TITLE
MSEloss by default uses 'mean' as reduction method

### ISSUE THREAD

**Issue by akiori** [jinglescode/time-series-forecasting-pytorch#13]
MSEloss by default uses 'mean' as reduction method, so I think `epoch_loss += (loss.detach().item() / batchsize)` is incorrect

**Comment by Alex-177** [comment]
I came in to write the same thing.
Besides, now the loss is accumulating on every butch. More precisely, the loss / batchsize value is accumulated.
It seems to me more logical to output the loss value on the last batch of training.
